modules/cluster-logging-upgrading-logging.adoc

```diff
@@ -0,0 +1,148 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging.adoc
+
+[id="cluster-logging-about_{context}"]
+= Upgrading cluster logging
+
+To upgrade cluster logging from 4.1 to 4.2,  after upgrading the cluster 
+update the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+
+.Prerequisites
+
+* Upgrade cluster from 4.1 to 4.2
+
+* Make sure the clusterlogging status is healthy
++
+** All pods are ready.
+** ES is cluster healthy
+
+* Make sure the is't  updating automatically by OLM. 
+
+.Procedure
+
+. Upgrade the Elasticsearch Operator:
+
+.. From the web console, click *Operators* -> *Installed Operators*. 
+
+.. Select the `openshift-logging` Project.
+
+.. Click the *Elasticsearch Operator*.
+
+.. Click *Subscription* -> *Channel*. 
+
+.. In the Change Subscription Update Channel window, select *4.2* and click `Save`.
+
+.. Wait for a seconds, then click *Operators* -> *Installed Operators*. 
++
+The Elasticsearch Operator is shown a 4.2. For example:
++
+----
+Elasticsearch Operator
+4.2.0-201909201915 provided 
+by Red Hat, Inc
+----
+
+.. Ensure that the Elasticsearch pods are using the 4.2 image:
++
+----
+$ oc get pod -o yaml -n openshift-logging --selector component=elasticsearch |grep 'image:'
```

Comment:
https://bugzilla.redhat.com/show_bug.cgi?id=1756304#c2.  After elasticsearch-operators is upgraded to 4.2. The ose-oauth-proxy is v4.2.0-*,  the elasticearch5 is still v4.1.x-*.  The elasticsearch5 image will be upgraded to 4.2 after clusterlogging is upgraded.

Reply:
@anpingli Thank you for pointing this out, What if we move the verification steps to the end of the procedure? 
Update ESO using the web console
Update CLO using the web console
Verify the pod upgraded and are active using the CLI

Reply:
That is OK.